dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_2_fitting.rst

```diff
@@ -1,132 +1,225 @@
+The tests here in group 2 are concerned with the different types of fits 
+possible through the **Fitting** and **Sequential Fitting** tabs. In each test, 
+the fit should succeed without a crash - the results will be tested later, in 
+:ref:`Muon_Analysis_TestGuide_3_Results-ref`. You will be using all the results 
+from these tests as inputs for :ref:`Muon_Analysis_TestGuide_3_Results-ref` so 
+**do not** clear or close the interface.
 
 The master testing guide is located at :ref:`Muon_Analysis_TestGuide-ref`.
 
-Tests
-^^^^^
+-----------
+
+.. _test_1:
 
-Test 1: individual fit
+Test 1: Individual Fit
 ----------------------
-- On the *Home* tab, select instrument EMU and load ``EMU00020918.nxs``.
-- Go to *Data Analysis* tab. Expand the window if the function browser is too small (you can drag the bar underneath it to set relative sizes of the widgets).
-- The loaded dataset should be pre-selected: "20918" in Runs and "All Pairs" selected in the data table.
-- "Label" box and "Co-add/Simultaneous" radio buttons should be disabled
-- Click and drag blue lines on graph, check start/end times are updated.
-- Check the reverse - change start/end times and blue lines should be updated on graph.
-- Set up the interface to look like this. Note the non-default values for ``f0.Omega`` and ``f0.Tau``:
+
+**Time required 5-10 minutes**
+
+- Open **Muon Analysis** (*Interfaces* > *Muon* > *Muon Analysis*)
+- Go to the **Home** tab, select instrument **EMU** and load ``20918``
+- Go to the **Fitting** tab
+	- Right click the empty table area; Select **Add Function**
+	- Add a **Flat Background** (*Background* > *Flat Background*)
+	- Similarly, add **Abragam** (*Muon* > *MuonSpecific* > *Abragam*)
+	- Set ``Omega = 8.5`` and ``Tau = 0.4``
+	- Click **Fit**
+- The fit should look something like this:
 
 .. image:: /images/MuonAnalysisTests/fitting_test1.png
+	:align: center
+	:alt: fitting_test1.png
+	
+- In the bottom table, set ``Time Start = 1`` and ``Time End = 5``
+- Click **Fit** again
+- The fit curve should be updated with the new time constraints and look like 
+  this:	
+  
+.. image:: /images/MuonAnalysisTests/fitting_test1_start_end_time.png
   :align: center
-  :height: 500
+  :alt: fitting_test1_start_end_time.png
 
-- Fit the data. Graph should be updated.
-
-Test 2: sequential fit
+-----------
+  
+.. _test_2:
+  
+Test 2: Sequential Fit
 ----------------------
-- With everything set up as in the picture above, click *Fit/Sequential fit*. (Runs should still be set to "20918", a single run).
-- Dialog should appear. In the runs box **of the new dialog that appears**, type "20918-20" and hit Start.
-- Sequential fit of runs 20918, 20919, 20920 should happen one after the other.
 
+**Time required 5 minutes**
 
+- Following the set up from :ref:`test_1`, load runs ``20918-20`` and reset 
+  ``Time Start = 0.113`` and ``Time End = 15``
+- Go to the **Sequential Fitting** tab
+	- Click **Sequentially fit all**
```

Comment:
select each row and see that the plots update.
You can change the values in the rows and then click fit selected to just do that one fit.